index.html

```diff
@@ -1853,6 +1854,7 @@ <h2>
           required DOMString label;
           required PaymentCurrencyAmount amount;
           boolean pending = false;
+          PaymentItemType? type;
```

Comment:
No need for this to be null. Just have them not supply it if they don't want to set it (or, set it to undefined).